gradle-plugin/src/main/groovy/io/realm/gradle/RealmRobolectric.groovy

```diff
@@ -0,0 +1,35 @@
+package io.realm.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class RealmRobolectric implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+        project.task('downloadRealmRobolectric') {
+            doLast {
+                def targetZip = "${project.projectDir}/realm-robolectric-darwin-${Version.VERSION}.zip"
+                def targetLib = "${project.projectDir}/robolectricLibs/librealm-jni-darwin.dylib"
+
+                if (project.file(targetZip).exists() && project.file(targetLib).exists()) {
+                    println "librealm-jni-darwin.dylib is already exsit. Skip download the Realm Robolectric."
+                    return
+                }
+
+                project.download {
+                    src "http://static.realm.io/downloads/java/realm-robolectric-darwin-${Version.VERSION}.zip"
+                    dest "${project.projectDir}"
+                }
+
+                project.copy {
+                    from project.zipTree(project.file(targetZip))
+                    into "${project.projectDir}/robolectricLibs"
```

Comment:
Any reason we are putting this in a separate folder called `robolectrictLibs`. Doesn't there already exist a `lib` folder? This would also mean we don't have to manually modify the "java.library.path", right?